libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationSymbolManager.java

```diff
@@ -52,4 +53,9 @@ private void createSymbolFrom(SymbolOptions options) {
     Symbol symbol = symbolManager.create(options);
     mapMarkersSymbols.add(symbol);
   }
+
+  @UiThread
```

Comment:
Why do we only need the UiThread annotation here? Wouldn't this be needed for the other methods as well?

Reply:
because `AnnotationManager.onDestroy` is annotated as `@UiThread` as per [here](https://github.com/maplibre/maplibre-plugins-android/blob/8b05f4654b83769c7755df249ad1d5f7d49551da/plugin-annotation/src/main/java/org/maplibre/android/plugins/annotation/AnnotationManager.java#L359).

good point, will update the other calls in the hierarchy to also be annotated as `@UiThread`.